docs/intentReaderHandlers/RegisterEventHandler.md

```diff
@@ -0,0 +1,357 @@
+# RegisterEventHandler 
+
+## Overview
+
+RegisterEventHandler is invoked when the task specified in the intent has the value "registerEvent". This handler is used to register for the event in FCA.
```

Comment:
what does **Registers the event passed in params.**  mean ? Can this be changed to
FCA registers for the event received in the input command.

Reply:
updated